### PR TITLE
Adding Ques based on Python Explicit type coversion

### PR DESCRIPTION
**Pull request by TauqeerAhmad5201**
On the issue of #15, adding questions based on Explicit type conversion from Python.

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,14 @@
 const pythonQuiz = [
+  {
+    Question: "In Python, which Type casting method let users convert a specific data type into required data type?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:3]:
```suggestion
    Question: "In Python, which Type casting method lets users convert a specific data type into a required data type?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/python-quiz.ts (hunk 1 above)
Comment (TauqeerAhmad5201) [reply src/data/python-quiz.ts]: Resolved changes with commit id 2c287d15b28b50d3747c0578c00a8de6fe428ae3

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**